AsyncDisplayKit/ASVideoNode.h

```diff
@@ -18,7 +18,10 @@
 //    in an issue on GitHub: https://github.com/facebook/AsyncDisplayKit/issues
 
 @interface ASVideoNode : ASControlNode
+
+@property (atomic, strong, readwrite) NSURL *url;
```

Comment:
Because it's so easy to create an asset with a URL, I'm still not confident we should provide this.  If the user sets an asset, then accesses this URL, it should return something - but what if the asset is local?  Setting both, etc.  For now, I'd like to remove it in a followup, and we can discuss the merits of adding it in continued iteration + make sure it is fully integrated for all the edge cases (and a header comment here about expected usage etc)